ods_ci/tests/Tests/500__jupyterhub/test-pipChanges-not-permanent.robot

```diff
@@ -21,7 +21,7 @@ Verify pip Changes not permanent
     Stop JupyterLab Notebook Server
     Capture Page Screenshot
     Fix Spawner Status
-    Spawn Notebook With Arguments
+    Spawn Notebook With Arguments    image=s2i-minimal-notebook
```

Comment:
Can you change it for `image=minimal-notebook`?  This will make it compatible with ODH too